tests/hypothesis_strategies/targets.py

```diff
@@ -39,5 +40,8 @@ def numerical_targets(
     )
 
 
-targets = numerical_targets()
+binary_targets = st.builds(BinaryTarget)
```

Comment:
no testing of different choice values? (could be important to test the conversions are done right)

Reply:
done